client/app/components/ParameterValueInput.jsx

```diff
@@ -40,6 +40,7 @@ class ParameterValueInput extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      reg: /\/\*(.*)\*\//i,
```

Comment:
This regex does not need to be a state, you can move it up as a `const`:

```js
const REMOVE_COMMENTS_REGEX = /\/\*(.*)\*\//i;

function ParameterValueInput() {
  // then use it here
}

Reply:
And thinking about edge cases: what if someone had `/* */` as part of their actual values? 🙂

Reply:
> And thinking about edge cases: what if someone had `/* */` as part of their actual values? 🙂

In SQL, it't a comment. So it's ok!

Reply:
Yes, but there are also non-SQL data sources and in SQL it could be inside strings as well, e.g.: `select * from table where column = "{{ param }}"`

Reply:
> there are also non-SQL data sources

Oh, that's a really good point.

Reply:
To me, the `{ value, label }` syntax seems like it would provide the same capability, but also be easier to understand + less likely to go wrong.

Reply:
> To me, the `{ value, label }` syntax seems like it would provide the same capability, but also be easier to understand + less likely to go wrong.

I will pefer this PR.